lms/static/js/student_account/views/FormView.js

```diff
@@ -55,9 +56,12 @@
             render: function( html ) {
                 var fields = html || '';
```

Comment:
Nit: I'd change this to:

    var fieldsHtml = HtmlUtils.HTML(html || ''),

IMO it is best to wrap html strings as early as possible so that they can be safely passed around. You might consider changing the callers to pass HTML snippets rather than raw strings.